### PR TITLE
ENCD-4166 RIN internal action audit

### PR DESCRIPTION
**Pull request by gabdank**
Adding an audit for missing RIN in RNA libraries

### CODE REVIEW

File: src/encoded/audit/experiment.py [hunk 1]
```diff
@@ -2906,6 +2906,25 @@ def audit_library_RNA_size_range(value, system, excluded_types):
...
+    RNAs = ['RNA',
+            'polyadenylated mRNA',
+            'miRNA']
+
+    for rep in value['replicates']:
+        if (rep['status'] not in excluded_types and
+           'library' in rep and rep['library']['status'] not in excluded_types and
+           rep['library']['nucleic_acid_term_name'] in RNAs and
+           'rna_integrity_number' not in rep['library']):
+            detail = ('Metadata of RNA library {} lacks specification of ' +
```
Comment (caseylitton) [inline src/encoded/audit/experiment.py]: The + can be removed

File: src/encoded/audit/experiment.py [hunk 2]
```diff
@@ -2906,6 +2906,25 @@ def audit_library_RNA_size_range(value, system, excluded_types):
     return
 
 
+def audit_RNA_library_RIN(value, system, excluded_types):
+    '''
+    An RNA library should have a RIN specified.
+    '''
+
```
Comment (caseylitton) [inline src/encoded/audit/experiment.py]: Can we collapse this all onto one line?  
```
RNAs = ['RNA', 'polyadenylated mRNA', 'miRNA']
for rep in value['replicates']:
```
It's a pretty short list.  If there is a concern about needing to add elements to it in the future then
```
RNAs = [
    'RNA',
    'polyadenylated mRNA',
    'miRNA',
]
for rep in value['replicates']:
```
would be prefered.

File: src/encoded/audit/experiment.py (hunk 1 above)
Comment (gabdank) [reply src/encoded/audit/experiment.py]: fixed

File: src/encoded/audit/experiment.py (hunk 2 above)
Comment (gabdank) [reply src/encoded/audit/experiment.py]: fixed

**Pull request merged**